src/BinDeps.jl

```diff
@@ -558,4 +558,10 @@ include("show.jl")
 
 @Base.deprecate_binding shlib_ext Libdl.dlext
 
+
+# Definitions that should be evaluated upon initialization, not stored in the cache
+function __init__()
+    global const has_sudo = try success(`sudo -V`) catch err false end
```

Comment:
This shouldn't be  a `global const`  which can cause issues when used in __init__ for precompilation